.github/workflows/_linux_transformers.yml

```diff
@@ -299,36 +293,63 @@ jobs:
           xpu-smi discovery -y --json --dump -1
       - name: Sanity check installed packages
         run: |
-          source activate $CONDA_ENV_NAME
+          # Use latest pytest
+          pip install -U pytest pytest-timeout pytest-xdist pytest-shard
           # These checks are to exit earlier if for any reason Transformers
           # reinstalled torch packages back to CUDA versions (not expected).
           pip show torch | grep Version | grep xpu
           pip show torchaudio | grep Version | grep xpu
           pip show torchvision | grep Version | grep xpu
           python -c 'import torch; exit(not torch.xpu.is_available())'
-      - name: Run tests
+      - name: Run tests on ${{ needs.prepare.outputs.hostname }}
         run: |
-          source activate $CONDA_ENV_NAME
+          set -x
+          set +e
           cd transformers
-          python3 -m pytest -rsf --make-reports=$TEST_CASE --junit-xml=reports/$TEST_CASE.xml \
-            -k "${{ matrix.test.filter}}" ${{ matrix.test.cmd }} || true
+          xpu_list=($(echo ${ZE_AFFINITY_MASK} |sed 's/,/ /g'))
```

Comment:
These changes did not exist just yesterday. Why do we need them? I thought we have agreed that previous version was almost ready to go. I definitely has concerns with this new version. For example, output table became much bigger and complex: https://github.com/intel/torch-xpu-ops/actions/runs/17613758986?pr=1999. We don't need all these categories...